apps/lockfile-explorer/src/start.ts

```diff
@@ -103,7 +105,17 @@ function startApp(debugMode: boolean): void {
 
   app.get('/api/lockfile', async (req: express.Request, res: express.Response) => {
     const pnpmLockfileText: string = await FileSystem.readFileAsync(appState.pnpmLockfileLocation);
-    const doc = yaml.load(pnpmLockfileText);
+    const doc = yaml.load(pnpmLockfileText) as Lockfile;
+    const { packages, lockfileVersion } = doc;
+    if (packages && lockfileVersion.toString().startsWith('6.')) {
+      const updatedPackages: Lockfile['packages'] = {};
+      for (const dependencyPath in packages) {
+        if (Object.prototype.hasOwnProperty.call(packages, dependencyPath)) {
+          updatedPackages[convertLockfileV6DepPathToV5DepPath(dependencyPath)] = packages[dependencyPath];
```

Comment:
Would it be better to convert V5->V6 instead of vice-versa?

Reply:
Yes, I think this is better. I copied this logic from the pnpm source code

Reply:
Additionally, I think resolving dependencies in the browser is not very appropriate, as it limits our use of pnpm-related open-source libraries.